src/main/java/seedu/duke/storage/Storage.java

```diff
@@ -0,0 +1,111 @@
+package seedu.duke.storage;
+
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Objects;
+import java.util.Scanner;
+
+import seedu.duke.exception.DukeException;
+import seedu.duke.task.Deadline;
+import seedu.duke.task.Event;
+import seedu.duke.task.Task;
+import seedu.duke.task.TaskList;
+import seedu.duke.task.ToDo;
+
+/**
+ * The Storage class handles the creation of a file specified by the duke.txt file.
+ */
+public class Storage {
+    /* The filepath that the file should be created or read from. */
+    protected String filePath;
+
+    /**
+     * Creates a Storage object.
+     *
+     * @param filePath The filePath of whether the file should be created or read from.
+     */
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+
```

Comment:
Noticed the double empty lines here too!

Reply:
Great catch! Thank you!